srtcore/core.cpp

```diff
@@ -7519,7 +7551,7 @@ void CUDT::processCtrl(const CPacket &ctrlpkt)
         // XXX The use of this field hasn't been found; a field with the
         // same name is found in FileSmoother (created after CUDTCC from UDT)
         // and it's updated with the value of m_iSndCurrSeqNo upon necessity.
-        m_iLastDecSeq   = m_iSndCurrSeqNo;
+        //m_iLastDecSeq   = m_iSndCurrSeqNo;
```

Comment:
I would suggest a separate commit/PR to remove `CUDT::m_iLastDecSeq`.

Reply:
It had a bad luck to be near to these changes. OTOH it should've been removed a long time ago.

Reply:
This change is of little impact because this field isn't used. However, this change is important because it's then part of the sequence resetting functions.

Reply:
So, `m_iLastDecSeq` will later be used?

Reply:
Most likely, it won't. Just no proper research on that topic was ever done.

Reply:
What kind of research? `CUDT::m_iLastDecSeq` is just not used.

Reply:
As to whether the fact that it's not used is due to it has been moved (and no longer necessary) or removed by mistake.

Reply:
v1.3.4 also does not use `CUDT::m_iLastDecSeq`.

Reply:
I'd simply postpone this until some bigger cleanup activity when we are more free.

Reply:
We are wasting time. We are cleaning up today as much as we can. There will never be any dedicated time for code cleanup. And we'll be looking at this beautifully commented code for the rest of our lives. 😢

Reply:
Ok, you won me over this :D

Reply:
@ethouris No postponing with this type of issues during the whole review process!